curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62b30924c5e4ef0daba23b5e.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 62b30924c5e4ef0daba23b5e
+title: Step 42
+challengeType: 0
+dashedName: step-42
+---
+
+# --description--
+
+The border of the last fieldset element looks a little out of place. You can select the last element of a specific type using the `last-of-type` CSS pseudo-class, like this: 
+
+```
+p:last-of-type { }
+```
+
+That will select the last `p` element. Create a new selector that targets the last `fieldset` element and set its `border-bottom` to `none`.
+
+# --hints--
+
+You can use the `:last-of-type` pseudo-class to select only the last element of a certain type.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type'));
+```
+
+Your `fieldset:last-of-type` should have `border-bottom` set as `none`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type')?.borderBottom, 'none');
```

Comment:
```suggestion
const borderBottom = new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type')?.borderBottom;
assert(borderBottom === 'none' || borderBottom === 'medium none');
```


This test isn't passing in firefox - works in safari and chrome. Firefox gives a value of `medium none` - chrome and safari both give `none`. I made a suggestion that works - feel free to improve it if you want.